packages/editor-ui/src/components/Node.vue

```diff
@@ -389,6 +455,9 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 	}
 }
 
+.node-tooltip {
```

Comment:
nitpick - since this is a global css class, maybe something a little more specific `.node-trigger-tooltip` might help

Reply:
Class is changed to .node-trigger-tooltip